.github/workflows/deploy_development.yml

```diff
@@ -29,14 +29,13 @@ jobs:
       # strategy:
```

Comment:
Use matrix to run container with python 3.10

Reply:
Commented code removed because this container doesn't use any python